api/users/urls.py

```diff
@@ -6,6 +6,6 @@
     url(r'^$', views.UserList.as_view(), name=views.UserList.view_name),
     url(r'^(?P<user_id>\w+)/$', views.UserDetail.as_view(), name=views.UserDetail.view_name),
     url(r'^(?P<user_id>\w+)/nodes/$', views.UserNodes.as_view(), name=views.UserNodes.view_name),
-    url(r'^(?P<user_id>\w+)/registrations/$', views.UserRegistrations.as_view(), name=views.UserRegistrations.view_name)
-
+    url(r'^(?P<user_id>\w+)/node_logs/$', views.UserNodeLogs.as_view(), name=views.UserNodeLogs.view_name),
```

Comment:
We have `users/me/nodes` and `users/me/registrations`, but the logs here are shared between nodes and registrations, so the endpoint name is confusing. What about `users/me/logs`?